tests/unit/components/Navigator.spec.js

```diff
@@ -112,6 +112,12 @@ describe('Navigator', () => {
   beforeEach(() => {
     jest.clearAllMocks();
   });
+
+  it('renders a nav', () => {
```

Comment:
This assertion could have been part of the `it('renders the Navigator'` test. I dont see a reason to have it separate.

Reply:
Sounds good :)